### PR TITLE
add pirates challenge

### PR DESCRIPTION
**Pull request by LeoCBS**
Data pirates challenge description

### CODE REVIEW

File: datapirates/challenge.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Data Pirates challenge
+
+Welcome to Data Pirates challenge.
+
+
+## Scenario:
+
+The Internet Movie Database (IMDb) is an online database of information about music, movies, movies, TV shows and games and television games computer. [Wikipedia]. This challenge consist to collect information from this site and deliver it other format.
```
Comment (kamilash) [inline datapirates/challenge.md:8]: movies twice here :)

Comment (kamilash) [reply datapirates/challenge.md:8]: ...from this site and deliver it **in** other format.

File: datapirates/challenge.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * Get 500 titles from each genre
+ * Sorted by 'Rating'
+ * This app must run with only one command
+ * The output format is JSONL
+ * Write each genre in one output file
+
+
+Delivery:
+
+The code should be sent through github, with a usage documentation or something.
```
Comment (kamilash) [inline datapirates/challenge.md:22]: Suggestion: "...or something that helps to get your app running."

Comment (guilherme-santos) [reply datapirates/challenge.md:22]: How about `with documentation how to run it`. you don't want to guess how to run the code from someone else, right? :)

File: datapirates/challenge.md (hunk 1 above)
Comment (LeoCBS) [reply datapirates/challenge.md:8]: thanks :)

File: datapirates/challenge.md (hunk 2 above)
Comment (LeoCBS) [reply datapirates/challenge.md:22]: indeed

thanks for your suggestions :)

**Pull request merged**